src/controllers/commands/builtin/twitch/UpdateColor.cpp

```diff
@@ -31,7 +31,7 @@ QString updateUserColor(const CommandContext &ctx)
     if (user->isAnon())
     {
         ctx.channel->addMessage(makeSystemMessage(
-            "You must be logged in to use the /color command"));
+            "You must be logged in to use the /color command!"));
```

Comment:
One last ! to ` conversion left, then it looks ready to merge!
I missed this one in my last review, sorry!